manifests/kustomize/env/gcp/params.env

```diff
@@ -4,3 +4,4 @@ cacheDb=cachedb
 bucketName=yourGcsBucketName
 gcsProjectId=yourGcsProjectId
 gcsCloudSqlInstanceName=yourCloudSqlInstanceName
+cacheImage='google/cloud-sdk:alpine'
```

Comment:
Nit: Did you take this change from a different branch?

Reply:
good catch, yes i did.